### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by SnakeOverflow**
Mostly complete Spanish translation, just missing stuff from HomeRVAdapter.kt

### CODE REVIEW

**Comment by starry69** [comment]
Thank-you for your hard work! I'll have a look after sometime and will try to export strings resources for adapter aswell hopefully tomorrow.

**Comment by SnakeOverflow** [comment]
Thank you 
I'll translate those whenever there ready.

**Comment by starry69** [comment]
Hi there! I've [exported](https://github.com/Pool-Of-Tears/GreenStash/commit/65d1276f5bc9aedc39a19a11b9b39f535e0c1ade) string resources for all of the missing fields you had mentioned. You can add translations for them now.

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    <string name="locale_pref_header">Locales</string>
+    <string name="security_pref_header">Seguridad</string>
+    <string name="about_pref_header">Sobre</string>
+    <!--  Display Preferences -->
+    <string name="display_pref_title">Tema Predeterminado</string>
+    <string name="material_you_perf_title">Habilitar Material You</string>
+    <string name="material_you_perf_summary">Cambia tema de acuerdo a tu fondo de pantalla (Requiere Android 12+).</string>
+    <string name="material_you_unavailable">Perdon, parece que Material You no esta soportado en su dispositivo o version de android.</string>
+    <!-- Locale Preferences -->
+    <string name="currency_pref_title">Moneda Predeterminada</string>`
```
Comment (starry69) [inline app/src/main/res/values-es/strings.xml:109]: Looks like you've added a (`) here by mistake.

**Comment by starry69** [comment]
Thank-you for your contribution! Merging.

File: app/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (SnakeOverflow) [reply app/src/main/res/values-es/strings.xml]: on line 109? just fixed it.

**Pull request merged**